specification/src/main/asciidoc/platform/Profiles.adoc

```diff
@@ -44,35 +44,35 @@ rules of the Java Community Process. Typically, a proposal to create a
 new profile, or to revise an existing one, will be submitted as a Java
 Specification Request (JSR). Once the JSR is approved, an expert group
 will be formed and conduct work as dictated by the process. The JSR for
-a profile must mention the version of the Java EE Platform that it
+a profile must mention the version of the Jakarta EE Platform that it
```

Comment:
This section needs some work.  We are no longer following the JCP processes.  We're following the Jakarta EE Specification Process.  And, we use Jakarta Specification Projects (within Eclipse) to define new specifications (or profiles).